alpha-blog/app/views/articles/index.html.erb

```diff
@@ -1,26 +1,4 @@
 <h1 class="text-center mt-4">Listing all articles</h1>
 
-<div class="container">
-  <% @articles.each do |article| %>
-    <div class="row justify-content-md-center">
-      <div class="col-8 mt-4">
-        <div class="card text-center shadow mb-5 bg-white rounded">
-          <div class="card-header font-italic">
-            by <%= article&.user&.username %>
-          </div>
-          <div class="card-body">
-            <h5 class="card-title"><%= link_to article.title, article_path(article), class: "text-success" %></h5>
-            <p class="card-text"><%= truncate(article.description, length: 100) %></p>
-            <a href="#"><%= link_to 'View', article_path(article), class: "btn btn-outline-success" %></a>
-            <a href="#"><%= link_to 'Edit', edit_article_path(article), class: "btn btn-outline-primary" %></a>
-            <a href="#"><%= link_to 'Delete', article_path(article), method: :delete, data: { confirm: "Are you sure?"}, class: "btn btn-outline-danger" %></a>
-          </div>
-          <div class="card-footer text-muted">
-            <small>Created <%= time_ago_in_words(article.created_at) %> ago, edited <%= time_ago_in_words(article.updated_at) %> ago</small>
-          </div>
-        </div>
-      </div>
-    </div>
-  <% end %>
-</div>
+<%= render 'article', articles: @articles %>
```

Comment:
Are you 10000% sure this one works without error?

Reply:
Yea it shows all the articles at `/articles`

Reply:
![image](https://user-images.githubusercontent.com/67599130/119687727-fffca600-be79-11eb-99cf-d88cee9f6144.png)

Reply:
where is `@articles` defined?